protobuf/coreapi/api.proto

```diff
@@ -13,78 +13,13 @@ option go_package = "github.com/mesg-foundation/core/protobuf/coreapi";
 //
 // The source file of this API is hosted on [GitHub](https://github.com/mesg-foundation/core/blob/master/protobuf/coreapi/api.proto).
 service Core {
-  // Subscribe to a stream that listens for events from a service.
-  rpc ListenEvent (ListenEventRequest) returns (stream EventData) {}
-
-  // Execute a service's task through Core.
-  rpc ExecuteTask (ExecuteTaskRequest) returns (ExecuteTaskReply) {}
```

Comment:
This API needs to stay, for now, we don't have any binding for that one

Reply:
I think this api should be moved to execution where it belongs